src/base/core/owl_computation_eval.ml

```diff
@@ -53,7 +53,7 @@ module Make (A : Ndarray_Mutable) = struct
         | Reshape shape                               -> _eval_map_01 x (fun ~out x -> A.reshape_ ~out x.(0))
         | Reverse                                     -> _eval_map_01 x (fun ~out x -> A.reverse_ ~out x.(0))
         | Tile repeats                                -> _eval_map_00 x (fun x -> A.tile x.(0) repeats)
-        | Repeat (axis, repeats)                      -> _eval_map_00 x (fun x -> A.repeat ~axis x.(0) repeats)
+        | Repeat repeats                              -> _eval_map_00 x (fun x -> A.repeat x.(0) repeats)
```

Comment:
this is not correct. but since you are not very familiar with computation graph, so it is fine, i will fix it later.